packages/flutter_tools/test/commands.shard/hermetic/pub_get_test.dart

```diff
@@ -181,6 +207,7 @@ class FakePub extends Fake implements Pub {
   FakePub(this.fileSystem);
 
   final FileSystem fileSystem;
+  List<String>? expectedArguments;
```

Comment:
wow, this is not pretty :see_no_evil: 

But I guess it's a way to test things.